active/cve-2019-5418.js

```diff
@@ -26,21 +26,33 @@ data from the server file system.'
     var wascId = 33 // Path Traversal
     var url = msg.getRequestHeader().getURI().toString();
 
-    msg = msg.cloneRequest();
-    msg.getRequestHeader().setHeader("Accept", "../../../../../../../../etc/passwd{{")
+    var msg2 = msg.cloneRequest();
+    msg2.getRequestHeader().setHeader("Accept", "../../../../../../../../etc/passwd{{")
 
-    as.sendAndReceive(msg, false, true);
+    as.sendAndReceive(msg2, false, true);
 
     var re = /root:.:0:0/g
-    var body = msg.getResponseBody().toString()
+    var body = msg2.getResponseBody().toString()
 
     if (re.test(body)) {
         as.raiseAlert(alertRisk, alertConfidence, alertTitle, alertDesc, 
-          url, '', '', alertInfo, alertSolution, body.match(re), cweId, wascId, msg);
+          url, '', '', alertInfo, alertSolution, body.match(re), cweId, wascId, msg2);
```

Comment:
Should the evidence be changed to just the first match like in the added `raiseAlert`?

Reply:
I could change it, but I don't think there can be multiple root entries in passwd

Reply:
Right, but the array when converted to string will still have the `[]`?

Reply:
Hmmm it worked fine for me, but it doesn't hurt to change it. In testing I had hit a comma seperated list while working on the new addition. Tweak coming.